src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -0,0 +1,91 @@
+package nextstep.subway.domain;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class Section extends BaseEntity {
+
+    @Id
+    @Column(name = "section_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "line_id", nullable = false)
+    private Line line;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "up_station_id", nullable = false)
+    private Station upStation;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "down_station_id", nullable = false)
+    private Station downStation;
+    private long distance;
+
+    protected Section() {
+
+    }
+
+    public Section(Line line, Station upStation, Station downStation, long distance) {
+        this.line = line;
+        this.upStation = upStation;
+        this.downStation = downStation;
+        this.distance = distance;
+    }
+
+    public List<Station> getStations() {
+        ArrayList<Station> stations = new ArrayList<>();
+        stations.add(upStation);
+        stations.add(downStation);
+        return stations;
+    }
+
+    public long getDistance() {
+        return distance;
+    }
+
+    public boolean isShortDistance(Section section) {
+        if (distance <= section.getDistance()) {
```

Comment:
```suggestion
        return distance <= section.getDistance()
```
그냥 이렇게만 해줘도 될 것 같아요..!